src/tip/cache.py

```diff
@@ -0,0 +1,24 @@
+import os
+import shutil
+
+
+CACHE_DIR = "/tmp/tip/cache"
```

Comment:
By default if user A writes to this dir, then user B can't read it. For such purposes we created '/tmp/shared' dir that can be accessed from different users. But in context of `tip` (if we want it to be public) there should be more general configurable solution that works also with default settings and is disabled by default (as we have very specific use-case).